app/data/models/returns.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (sequelize, DataTypes) => {
+  const returns = sequelize.define('returns', {
```

Comment:
Suggest calling the model definition and the variable `return`.  As this represents a single entity rather than a collection.

Also is consistent with other repos.

Reply:
I can't call the variable return, as 'return' is not allowed as a declaration name or at least that's what vs code is telling me

Reply:
Ah of course.  Ignore that.  Though still see my other comment about ditching the variable anyway